src/components/image_upload.tsx

```diff
@@ -214,12 +228,26 @@ export class ImageUpload extends React.Component<ImageUploadProps, ImageUploadSt
         );
     }
 
+    private _renderFileSizeExceedsMessage(): JSX.Element {
+        const { fileSizeExceedsMessage, size } = this.props;
+        return (
+            <div className="image-upload__warning" style={{ width: size }}>
```

Comment:
The class selector `image-upload__warning` seems to be unused. Is the `style` attribute necessary or could this be done using CSS instead?

Reply:
The attribute `style` is necessary here for specifying the size, because using `<ImageUploadField />` requires the developer to specify the size of this field. The size varies from one image upload field to another, therefore it cannot be specified in CSS.